knative-operator/deploy/resources/kourier/kourier.yaml

```diff
@@ -0,0 +1,289 @@
+#apiVersion: v1
```

Comment:
Generally speaking: Can we make this file to be a plain Kourier release manifest taken straight from their upstream? And add the version to the filename?

Just to make it easy for us to upgrade that manifest if needed and so we don't have to think about what we need to change and stuff. The namespace for instance will be injected by the code, so we can leave this to what upstream defines as well.

Reply:
Yes. Done.